src/tests/system/lib/sssd/utils/authentication.py

```diff
@@ -3,17 +3,21 @@
 from __future__ import annotations
 
 from datetime import datetime
-from typing import Any
+from typing import TYPE_CHECKING
 
 from pytest_mh import MultihostHost, MultihostUtility
 from pytest_mh.ssh import SSHClient, SSHProcessResult
 
+if TYPE_CHECKING:
+    from pytest_mh.utils.fs import LinuxFileSystem
```

Comment:
I wonder, why is it required to guard it with `TYPE_CHECKING`? What is the import cycle?

Reply:
imported the LinuxFileSystem.